_sass/_components/_logo-bar.scss

```diff
@@ -101,11 +82,17 @@ $logo-bar-scale: 14px;
           max-height: $logo-bar-scale * 1.25;
         }
 
-        &[src*="zipkin_small.png"] {
+        &[src*="grpc"] {
```

Comment:
Sounds good, @reimertz. Would you like to make these changes as a separate PR? I'm fine either way.